rxandroidble/src/main/java/com/polidea/rxandroidble/exceptions/BleIllegalOperationException.java

```diff
@@ -0,0 +1,27 @@
+package com.polidea.rxandroidble.exceptions;
+
+import android.support.annotation.RestrictTo;
+
+import java.util.UUID;
+
+/**
+ * This exception is thrown when a non-supported operation has been requested upon a characteristic, eg. write operation on a
+ * characteristic with only read property.
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
the annotation should be on the constructor only. the class is meant to be used in the API